tests/cephfs/cephfs_bugs/test_clone_delete_with_FS_fail.py

```diff
@@ -0,0 +1,197 @@
+        fs_util.prepare_clients(clients, build)
+        fs_util.auth_list(clients)
+        log.info("checking Pre-requisites")
+        if len(clients) < 1:
+            log.info(
+                f"This test requires minimum 1 client nodes.This has only {len(clients)} clients"
+            )
+            return 1
+        default_fs = "cephfs_clone"
+        mounting_dir = "".join(
+            random.choice(string.ascii_lowercase + string.digits)
+            for _ in list(range(10))
+        )
+        client1 = clients[0]
+        fs_details = fs_util.get_fs_info(client1, fs_name=default_fs)
+        if not fs_details:
+            fs_util.create_fs(client1, default_fs)
+        subvolumegroup_list = [
+            {"vol_name": default_fs, "group_name": "subvolgroup_clone_cancel_1"}
+        ]
+        for subvolumegroup in subvolumegroup_list:
+            fs_util.create_subvolumegroup(client1, **subvolumegroup)
+        subvolume = {
+            "vol_name": default_fs,
+            "subvol_name": "subvol_clone_cancel",
+            "group_name": "subvolgroup_clone_cancel_1",
+            "size": "5368706371",
+        }
+        fs_util.create_subvolume(client1, **subvolume)
+        log.info("Get the path of sub volume")
+        subvol_path, rc = client1.exec_command(
+            sudo=True,
+            cmd=f"ceph fs subvolume getpath {default_fs} subvol_clone_cancel subvolgroup_clone_cancel_1",
+        )
+        kernel_mounting_dir_1 = f"/mnt/cephfs_kernel{mounting_dir}_1/"
+        mon_node_ips = fs_util.get_mon_node_ips()
+        fs_util.kernel_mount(
+            [clients[0]],
+            kernel_mounting_dir_1,
+            ",".join(mon_node_ips),
+            sub_dir=f"{subvol_path.strip()}",
+            extra_params=f",fs={default_fs}",
+        )
+        client1.exec_command(
+            sudo=True,
+            cmd=f"python3 /home/cephuser/smallfile/smallfile_cli.py --operation create --threads 10 --file-size 400 "
+            f"--files 100 --files-per-dir 10 --dirs-per-dir 2 --top "
+            f"{kernel_mounting_dir_1}",
+            long_running=True,
+        )
+        snapshot = {
+            "vol_name": default_fs,
+            "subvol_name": "subvol_clone_cancel",
+            "snap_name": "snap_1",
+            "group_name": "subvolgroup_clone_cancel_1",
+        }
+        fs_util.create_snapshot(client1, **snapshot)
+        clone_list = [
+            {
+                "vol_name": default_fs,
+                "subvol_name": "subvol_clone_cancel",
+                "snap_name": "snap_1",
+                "target_subvol_name": f"clone_{x}",
+                "group_name": "subvolgroup_clone_cancel_1",
+            }
+            for x in range(1, 8)
+        ]
+        rmclone_list = [
+            {"vol_name": default_fs, "subvol_name": f"clone_{x}"} for x in range(1, 8)
+        ]
+        for i in range(1, 10):
+            log.info(f"Iteration {i} ")
```

Comment:
I believe in the interest of time, the logs were captured for 4 iterations. please ack if that is the case and we are good to run for 10 iterations.

Reply:
yes Neha